charts/terranetes-controller/values.yaml

```diff
@@ -71,11 +71,16 @@ controller:
   # indicate we create the watcher jobs in user namespace, these allow users
   # to view the terraform output
   enableWatchers: true
+  ## Indicates we should forgo the controller registering it's own webhooks and allowing
+  ## helm to manage the webhooks for us
+  enableHelmWebhookRegistration: false
+  ## indicates if the controller should register the valdiation and mutation webhooks
+  ## for the Configuration, Revision and CloudResource resources
+  enableControllerWebhookRegistration: true
```

Comment:
Keeping the current behavior, so to enable it you need to overload the helm values

```yaml
enableControllerWebhookRegistration: false
enableHelmWebhookRegistration: true
```